extensions/wikia/AdEngine/js/lookup/services.js

```diff
@@ -63,56 +61,17 @@ define('ext.wikia.adEngine.lookup.services', [
 				slotTargeting[key] = amazonParams[key];
 			});
 		}
-	}
-
-	// Copied from AdLogicPageParams
-	// No longer needed when AmazonOld is removed
-	function decodeLegacyDartParams(dartString) {
-		var params = {},
-			kvs,
-			kv,
-			key,
-			value,
-			i,
-			len;
-
-		log(['decodeLegacyDartParams', dartString], 'debug', logGroup);
-
-		if (typeof dartString === 'string') {
-			kvs = dartString.split(';');
-			for (i = 0, len = kvs.length; i < len; i += 1) {
-				kv = kvs[i].split('=');
-				key = kv[0];
-				value = kv[1];
-				if (key && value) {
-					params[key] = params[key] || [];
-					params[key].push(value);
-				}
-			}
-		}
-
-		return params;
-	}
 
-	// No longer needed when AmazonOld is removed
-	function extendPageTargeting(pageTargeting) {
-		var amazonParams;
+		if (oxBidder && oxBidder.wasCalled() && Object.keys) {
```

Comment:
This code is very similar to the code used for Amazon. I wonder if we could extract it somehow and make it less DRY.